spec/cc/analyzer/include_paths_builder_spec.rb

```diff
@@ -42,7 +42,7 @@ module CC::Analyzer
       end
 
       it "keeps sibling files and directories in include_paths" do
-        %w[subdir/ trackable.rb].each do |trackable_file_or_directory|
+        %w[subdir trackable.rb].each do |trackable_file_or_directory|
```

Comment:
Gotcha, wasn't sure if there was a reason for that or not. I'll change it back.

The engine should probably check for itself too though.